### ISSUE TITLE
Vehicle status text which says "SEE CONSOLE" should be changed due to confusion

### ISSUE THREAD

**Issue by DonLakeFlyer** [PX4/PX4-Autopilot#16044]
Look here: https://discuss.px4.io/t/qgc-console-error-messages-meanings-causes/19228 for example confusion. QGC has a "Console" for it's logs. So users will be confused by this easily. I'm not sure what the correct terminology should be but CONSOLE it isn't.

**Comment by Antiheavy** [comment]
agree, we see this confusion a lot with users.  Maybe "See message window" or "See alerts" or whatever that thing is actually called.
